app/helpers/application_helper.rb

```diff
@@ -282,12 +289,12 @@ def crawlable_meta_data(opts = nil)
     opts[:twitter_summary_large_image] =
       get_absolute_image_url(opts[:twitter_summary_large_image]) if opts[:twitter_summary_large_image].present?
 
-    # Add opengraph & twitter tags
     result = []
     result << tag(:meta, property: 'og:site_name', content: SiteSetting.title)
     result << tag(:meta, property: 'og:type', content: 'website')
 
-    result = generate_twitter_card_metadata(opts, result)
+    generate_twitter_card_metadata(result, opts)
```

Comment:
Don't we still need the `result` variable here?

Reply:
It was confusing previously, it was amending the input and also returning it, maybe we should rename and add a ! for clarity